receiver/awsecscontainermetricsreceiver/go.mod

```diff
@@ -3,9 +3,8 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awsecs
 go 1.14
 
 require (
-	github.com/census-instrumentation/opencensus-proto v0.2.1
-	github.com/stretchr/testify v1.5.1
-	go.opentelemetry.io/collector v0.5.0
+	github.com/census-instrumentation/opencensus-proto v0.3.0
+	github.com/stretchr/testify v1.6.1
+	go.opentelemetry.io/collector v0.8.0
```

Comment:
I see. Somehow we missed the updating of this dependency earlier?

Reply:
Was merged after, and PR was very old